### PR TITLE
Update information about dependecies in `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by sobolevn**
Previous information was clearly outdated.

### CODE REVIEW

**Comment by sobolevn** [comment]
@Mariatta looks like new CI works fine 🎉 
https://github.com/python/the-knights-who-say-ni/runs/5307465742?check_suite_focus=true

**Comment by Mariatta** [comment]
Should all 4 checks be required? Or just the latest one. Couldn't remember which one did we have as required check before.

**Comment by ewdurbin** [comment]
That check doesn't exist as a status result for selection yet (nor do any others).

**Comment by Mariatta** [comment]
Hmm, weird. Was there anything special you did to make the status appear on this PR @sobolevn?

**Comment by sobolevn** [comment]
I can't see this repo's admin panel, but my fork allows me to do the same thing locally:
Снимок экрана 2022-02-23 в 21 24 38
Снимок экрана 2022-02-23 в 21 24 45

**Comment by ewdurbin** [comment]
nothing here:

Screen Shot 2022-02-23 at 1 34 24 PM

**Comment by hugovk** [comment]
Try typing "Python" instead of "test"? 
Because of https://github.com/python/the-knights-who-say-ni/pull/310/commits/923fbe360ab131fe8b6ef7f9fec269e51b74f89b

**Comment by ewdurbin** [comment]
oh yep, there we go.

**Comment by sobolevn** [comment]
It works! Thanks!
Снимок экрана 2022-02-23 в 21 57 40

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ Work done surrounding this project is governed by the
 
 ## Dependencies
 The `requirements.txt` file is frozen to the currently tested
-versions of `requirements.base`. To update, create a venv,
-run `python3 -m pip install -r requirements.base`, run all tests,
-and then update `requirements.txt` with the output from `pip freeze`.
+versions of `requirements.in`. To update, create a venv,
+install [`pip-tools`](https://pypi.org/project/pip-tools)
+run `python3 -m pip-sync`, run all tests, and then update
```
Comment (Mariatta) [inline CONTRIBUTING.md:11]: `python3 -m pip-sync` doesn't work for me. I needed to do just `pip-sync`.

Maybe easier if these instructions are in bullet points?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,9 +6,12 @@ Work done surrounding this project is governed by the
 
 ## Dependencies
 The `requirements.txt` file is frozen to the currently tested
-versions of `requirements.base`. To update, create a venv,
-run `python3 -m pip install -r requirements.base`, run all tests,
-and then update `requirements.txt` with the output from `pip freeze`.
+versions of `requirements.in`. To update, create a venv,
+install [`pip-tools`](https://pypi.org/project/pip-tools)
+run `python3 -m pip-sync`, run all tests, and then update 
+`requirements.txt` with `python3 -m pip-compile --upgrade` command.
```
Comment (Mariatta) [inline CONTRIBUTING.md:12]: Tested locally, the command `python3 -m pip-compile --upgrade` doesn't seem to work. I had to do just `pip-compile --upgrade`.

**Pull request closed without merging**